support-frontend/assets/pages/[countryGroupId]/events/router.tsx

```diff
@@ -1,21 +1,10 @@
-import ophan from 'ophan';
 import { createBrowserRouter, RouterProvider } from 'react-router-dom';
-import * as abTest from 'helpers/abTests/abtest';
-import { Country, CountryGroup } from 'helpers/internationalisation';
+import { setUpTrackingAndConsents } from 'helpers/page/page';
 import { renderPage } from 'helpers/rendering/render';
-import { trackAbTests } from 'helpers/tracking/ophan';
 import { geoIds } from 'pages/geoIdConfig';
 import { Events } from './events';
 
-const countryId = Country.detect();
-const countryGroupId = CountryGroup.detect();
-const participations = abTest.init({
-	countryId,
-	countryGroupId,
-});
-ophan.init();
-trackAbTests(participations);
-
+setUpTrackingAndConsents();
```

Comment:
This function has a few bits of logic specific to Contributions (such as the call to `getAmountsTestVariant`), this doesn't cause any issues?